org.eclipse.editorconfig.ui/src/main/java/org/eclipse/editorconfig/ui/internal/EditorConfigVisitor.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017 Nathan Jones, Jackson Bailey
```

Comment:
That fixes it, they work with and without my changes now. If want to you merge those into master I can rebase or if you don't like rebases I can just cherry-pick it.